dockstore-webservice/src/main/java/io/dockstore/webservice/languages/NextFlowHandler.java

```diff
@@ -81,21 +84,34 @@ public Entry parseWorkflowContent(Entry entry, String filepath, String content,
     @Override
     public Map<String, SourceFile> processImports(String repositoryId, String content, Version version,
         SourceCodeRepoInterface sourceCodeRepoInterface, String filepath) {
+        // FIXME: see{@link NextflowUtilities#grabConfig(String) grabConfig} method for comments on why
+        // we have to look at imports in this crummy way
+        Pattern pattern = Pattern.compile("(?i)(?m)^[ \t]*includeConfig(.*)");
```

Comment:
This is probably not called that frequently, but if it is, it might be a tiny performance improvement to make this a a static that's only instantiated once.